src/modules/presets.js

```diff
@@ -14,6 +16,19 @@ function configPrefix(include) {
   };
 }
 
+function getCallerAppVersion(pwd = __dirname) {
+  try {
+    const callerModulePackageJSON = pwd.includes('node_modules') ?
+    `${pwd.split(`${path.sep}node_modules`)[0]}${path.sep}package.json` : '';
```

Comment:
This one liner is a bit difficult to read, can you please try to enhance the readability